code/modules/mob/living/inventory.dm

```diff
@@ -58,6 +59,16 @@
 		if(istype(I))
 			I.on_active_hand()
 
+// Defer proc for the sake of delimbing root limbs with multiple graspers (serpentid)
+/mob/living/proc/queue_hand_rebuild()
+	set waitfor = FALSE
+	if(!pending_hand_rebuild)
+		pending_hand_rebuild = TRUE
+		sleep(1)
```

Comment:
Not a huge fan of this but until we get a queue/unique-queue subsystem to reuse, it'll do.